packages/calypso-e2e/src/browser-helper.ts

```diff
@@ -109,17 +109,11 @@ export function targetGutenbergEdge(): boolean {
  * @returns {LaunchOptions} Logger configuration.
  */
 export async function getDefaultLoggerConfiguration(): Promise< LaunchOptions > {
-	const { testPath } = getState() as { testPath: string };
-	const sanitizedTestFilename = path.basename( testPath, path.extname( testPath ) );
-	const resultsPath = path.join( process.cwd(), 'results' );
-	await mkdir( resultsPath, { recursive: true } );
-	const tempDir = await mkdtemp( path.join( resultsPath, sanitizedTestFilename + '-' ) );
-
 	return {
 		logger: {
 			log: async ( name: string, severity: string, message: string ) => {
 				await appendFile(
-					path.join( tempDir, 'playwright.log' ),
+					path.join( await getArtifactDir(), 'playwright.log' ),
```

Comment:
This should work, but out of curiosity, was it being fussy when trying to take this in as a parameter?

Reply:
I don't recall correctly anymore, but I suspect it was my slightly overzealous refactor that was the problem.

Reply:
If it works with the current state of the refactor, it might be worth making that tweak. I was looking at the hooks code, and since we call `getArtifactDir()` before we call this function, we actually end up calling that twice!

Otherwise, we can also punt that to later and just get this merged :)

Reply:
Yeah, it looks like that's the case, though it doesn't seem to do anything unexpected like create two directories. 

I have created a new task for the hook/artifact refactor - #56449.